defines/defines.go

```diff
@@ -35,6 +35,7 @@ const (
 	BatchIntervalUploadStats        = 0.95
 	ChannelInBufferSize             = 800
 	ChannelOutBufferSize            = 800
+	DefaultConfigFile               = "/etc/arachne/arachne.yaml"
```

Comment:
imo, a better default would be a cwd/pwd-relative "arachne.yaml" or "config.yaml", leaving any absolute paths to be specified outside of code; in other words, default to "only/maybe useful for local development, but all distributors/integrators want to set the flag".